src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -692,7 +692,12 @@ PlusStatus vtkPlusWinProbeVideoSource::InternalConnect()
   std::this_thread::sleep_for(std::chrono::milliseconds(100));
   if(m_Mode == Mode::M)
   {
-    this->SetMModeEnabled(true);
+    SetMIsEnabled(true);
```

Comment:
Calling `this->SetMModeEnabled(true)` would avoid code duplication. Please amend this commit.

Reply:
Do I remove the check `if(Connected)` within `setMModeEnabled`? Or, if I moved `this->SetMModeEnabled(true)`; to `InternalStartRecording()` is it okay to apply all M-Mode settings when unfreezing probe?

Reply:
I forgot that during `InternalConnect`, connected is still `false`. Leave the code as-is.